redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx

```diff
@@ -96,6 +99,24 @@ const Recommendation = ({
     history.push(Pages.workbench(instanceId))
   }
 
+  const toggleHide = (event: React.MouseEvent) => {
+    event.stopPropagation()
+    dispatch(
+      updateLiveRecommendation(
+        id,
+        { hide: !hide },
+        (_, { hide, name }) => sendEventTelemetry({
+          event: TelemetryEvent.INSIGHTS_RECOMMENDATION_HIDE,
+          eventData: {
+            databaseId: instanceId,
+            action: hide ? 'hide' : 'unhide',
+            name,
```

Comment:
Same telemetryEvent instead name. (I guess i can add additional function to replace name with telemetryEvent or you can do it)

Reply:
changed
good catch!